src/uct/api/uct.h

```diff
@@ -2626,6 +2630,41 @@ UCT_INLINE_API ucs_status_t uct_ep_am_short(uct_ep_h ep, uint8_t id, uint64_t he
 }
 
 
+/**
+ * @ingroup UCT_AM
+ * @brief Short send operation.
+ *
+ * This routine sends a message using @ref uct_short_protocol_desc "short" protocol.
+ * The input data in @a iov array of @ref ::uct_iov_t structures is sent to remote
+ * side ("gather output"). Buffers in @a iov are processed in array order.
+ * This means that the function completes iov[0] before proceeding to
+ * iov[1], and so on.
```

Comment:
* What does `completes` mean here ? I guess some ordering guarantee and not completion ? Are you waiting for a remote ack before you send the next element ?
* What `gather output` actually means (I know what it means but you have to describe). What happens if receiver has not enough memory to accept the vector ?

Reply:
Yes, you're right. `completes` is not applicable here.
`gather output` is only about that we get iovs into contiguous buffer on the receiver.
The size of the message to be sent cannot exceed a predefined size. So the total size of data in iovs should fit into the limit. It should be mentioned in the description of the method. I will add this.